src/components/AdminPage/TopMenu.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Menu } from 'semantic-ui-react';
+import maiLogo from '../Logo';
+
+
+function TopMenu() {
+  return (
+    <Menu className="TopMenu" position="top" borderless>
+      <Menu.Item className="LogoMenuItem">
+        <div>
+          {maiLogo}
+        </div>
```

Comment:
Why do you need the div?

Reply:
The div could be removed i don't see a need for it could be a mistake.